app/helpers/UpscanMessageHelper.scala

```diff
@@ -29,7 +33,7 @@ object UpscanMessageHelper {
     }
   }
 
-  def getUploadFailureMessage(errorCode: String): String = {
+  def getUploadFailureMessage(errorCode: String)(implicit messages: Messages): String = {
```

Comment:
Do you need the implicit here?

Reply:
Yes  the 'getMessage' method uses it as an implicit parameter.

` if (total == 1) {
      stringAsHtml(getMessage(msgForSingular, arg))
    } else {
      stringAsHtml(getMessage(msgForPlural, arg))
    }`

Reply:
In this method we aren't using getMessage - we are just returning a message key without using the messages API

Reply:
👍 got it Thanks.